tests/playwright-test/test-step.spec.ts

```diff
@@ -1289,3 +1294,144 @@ test('should not propagate nested hard errors', async ({ runInlineTest }) => {
     },
   ]);
 });
+
+test('should box step no-box', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'reporter.ts': stepHierarchyReporter,
+    'playwright.config.ts': `module.exports = { reporter: './reporter', };`,
+    'a.test.ts':
+    ` /*1*/ import { test, expect } from '@playwright/test';
+      /*2*/ test('fail', async () => {
+      /*3*/   await test.step('boxed step', async () => {
+      /*4*/     expect(1).toBe(2);
+      /*5*/   });
+      /*6*/ });
+    `
+  }, { reporter: '' });
+
+  expect(result.exitCode).toBe(1);
+  const objects = result.outputLines.map(line => JSON.parse(line));
+  expect(objects).toEqual([
+    {
+      category: 'hook',
+      title: 'Before Hooks',
+    },
+    {
+      category: 'test.step',
+      error: expect.stringContaining('a.test.ts:4:27'),
+      location: {
+        column: 26,
+        file: 'a.test.ts',
+        line: 3,
+      },
+      steps: [
+        {
+          category: 'expect',
+          error: expect.stringContaining('a.test.ts:4:27'),
+          location: {
+            column: 27,
+            file: 'a.test.ts',
+            line: 4,
+          },
+          title: 'expect.toBe',
+        },
+      ],
+      title: 'boxed step',
+    },
+    {
+      category: 'hook',
+      title: 'After Hooks',
+    },
+  ]);
+});
+
+test('should box step content', async ({ runInlineTest }) => {
+  const result = await runInlineTest({
+    'reporter.ts': stepHierarchyReporter,
+    'playwright.config.ts': `module.exports = { reporter: './reporter', };`,
+    'a.test.ts':
+    ` /*1*/ import { test, expect } from '@playwright/test';
+      /*2*/ test('fail', async () => {
+      /*3*/   await test.step('boxed step', async () => {
+      /*4*/     expect(1).toBe(2);
+      /*5*/   }, { box: 'content' });
+      /*6*/ });
+    `
+  }, { reporter: '' });
+
+  expect(result.exitCode).toBe(1);
+  const objects = result.outputLines.map(line => JSON.parse(line));
+  expect(objects).toEqual([
```

Comment:
I don't see any difference in the expectations between this test and the previous test, even though one has `box: 'content'`.